iAM-legacy/Simulation/Investments.cs

```diff
@@ -211,14 +232,44 @@ public void LoadBudgets()
         /// <param name="strYear"></param>
         /// <param name="hashAttributeValue"></param>
         /// <param name="priority"></param>
+        /// <param name="limits">Limits where treatments are split over multiple years</param>
         /// <returns></returns>
-        public string IsBudgetAvailable(float fAmount, String strBudget, String strYear, Hashtable hashAttributeValue,Priorities priority, out string budgetHash)
+        public string IsBudgetAvailable(float fAmount, String strBudget, String strYear, Hashtable hashAttributeValue,Priorities priority, List<ISplitTreatmentLimit> limits, string budgetLimitType, out string budgetHash, out ISplitTreatmentLimit splitTreatmentLimit)
         {
+            
+            splitTreatmentLimit = limits[0];
+
+            //Check priority.  If not possible just return null.
+            if (!(priority.IsAllSections || priority.Criteria.IsCriteriaMet(hashAttributeValue)))
+            {
+                budgetHash = "";
+                return null;
```

Comment:
Avoid returning null in API calls.

Reply:
Returns empty string.  The consumer of this function was asking for a budget that had money and met criteria.  Returning null was showing that no budget was available.  Return value was being checked IsStringNullEmptyWhitespace